### PR TITLE
Bunch of Game Additions

### PR DESCRIPTION
**Pull request by BurnerXD**
It's mostly good to go. If you want to merge that's fine, just let me know.
expansionsettings.gd
* Changes ApplyTweaks to applyRalphsTweaks
* Added applyCapitulizeTweaks
    * For tweaks I made for my game, includes more furry centric changes, maximum stat increases from traits, and lowered banditishuman chance.

expansiontalk.gd
* Buncha typo fixes (namely ellipsis), included kobold racial getMasterNoun.

expansionsetup.gd
* Fixed potential logic failure with adding to stats and reverse.
* Made logic for determining elements cleaner.
* Made logic to determine highest element.
* Adjusted logic for racial mental stat gains to separate, lowered absurd values.
* Added basic sections for new races within setRaceBonus_Ralph.
* Added backwards compatibility for weapons I believe.

globals.gd
* Added three new specializations. (Mage, Dancer, Warrior)
* Added correlating description arrays for new descriptions.
* Adjusted any logic related to races to include new races.

Image additions
* Include images for:
    * Serrated Dagger (It's shit)
    * Basic Staff (It's shit)
    * Dancer Spec Icon
    * Warrior Spec Icon
    * Mage Spec Icon
    * Desert Lizards (Not Implemented, credit to Rendrassa)
    * Swamp Lizards (Implemented, credit to Rendrassa)
    * Avali Shades 
    * Beastkin Bird Shades
    * Beastkin Mouse Shades
    * Beastkin Otter Shades
    * Beastkin Squirrel Shades
    * Gnoll Shades
    * Kobold Shades
    * Lizardfolk Shades
    * New Image for Mansion Top Bar

Mansion.tscn
* Adjusted the top bar so that it displays base stats that have been added to before modification. Integrated to update whenever necessary.

Sex Changes:
* 111assfingering.gd - Typo Fix related to asses.
* 112cunnilingus.gd - Removed unnecessary code.
* 121blowjob.gd - Added blowjob metrics. (Prolly a better way to do this.)
* 124footjob.gd - Fixed spacing typo in initiate.
* 125suckballs.gd - Added sex action.
* 208cowgirl.gd - Added sex action. (Credit to Darwin.w on Itch.io)
* 209reversecowgirl.gd - Added sex action. (Credit to Darwin.w on Itch.io)
* 210cowgirlanal.gd - Added sex action. (Credit to Darwin.w on Itch.io)
* 211revcowgirlanal.gd - Added sex action. (Credit to Darwin.w on Itch.io)
* 212tribadismonpenis.gd - Added sex action (Female givers only, needs more than 1).
* 222inserttailv.gd - Typo Fix.
* 222inserttaila.gd - Typo Fix.
* 265triplepen.gd -  Added sex action (3 givers with dicks to female).
* 507facefuck.gd - Added sex action.

dictionary.gd
* Removed Gnoll, Kobold, and Ogre from the hybriddict.

items.gd
* Added weaponbasicstaff.
* Added weaponserrateddagger.
* Gave every weapon a scaling attribute. It's very basic.
* Changed longsword to 2 strength requirement.

jobs&specs.gd
* Added dancer spec.
* Added warrior spec.
* Added mage spec.
* Typo fix on bodyguard.
* Added gnollhunt function. (For gnolls)
* Adjusted nurse to heal more when an avali is a nurse.
* Made dancers and mages make 1.2x more money when working for the mage order or entertainer jobs respectively.

outside.gd
* Adjusted logic on button disabling to be dynamic.
* Added the serrated dagger and basic staff to the wimborn market.
* Added logic to check if a mousekin is in the party when it comes to opening chests.

sexdescriptions.gd
* Added logic to handle new races and descriptions for sex related dialogue.

variables.gd
* Added scaling weapon damage attributes.
* Added ogre racial bonus as bonus health per point of endurance and bonus base health.

Mansion.gd
* Added outOfMansionJobs array.
* Added logic to handle racial bonuses for Gnolls and Giants (Giant's x3 food consumption applies to player too).
* Made it so a slave can gain the slutty trait after having gained devoted.
* Added logic to update base stats for new top bar in mansion.
* Added basic framework for specialization based abilities. Currently only supports one ability.

abilites.gd
* Added deadly blow, an ability that can only be unlocked if the slave is either a warrior or assassin.

assets.gd
* Added logic to support new races.
* Added support for baldness.
* Added Whims logic to obtaining names for slaves. 

constructor.gd
* Added logic to support new races in various arrays and through the file.

description.gd

description.gd
* Tied Hairstyles to Hairlength, so if someone is bald it won't display their hairstyle.
* Added description for over 100 beauty.
* Added baby description support for a bunch of new descriptions.
* Penises
    * Typo fix with penises.
    * Added reptilian penises.
    * Added bird penises.
    * Added rodent penises.
* New bodyshapes (giant, reptilian, reptilianshortstack, furryshortstack, avian, raptorshortstack).
* New beauty (gorgeous).
* New hairlength (bald).
* Adjusted straight hairstyle to be universally applied.
* New hairstlye (bald)
* Added scalecolor
* Added feathercolor
* New horns (manyhorned)
* New ears (short_reptilian, long_pointy_reptilian, frilled, none, long_round_reptilian, long_droopy_reptilian, wide_furry, avali)
* New skin (none)
* New tails (hyena, neko, otter, squirrel, reptilian, avali)
* New skincov (feathers_and_fur (this skincov uses both feathercolor and furcolor), fullscales, fullfeather)
* Adjusted feathers (includes feathercolor in description)
* New furcolor (blue, whiteandpink, 'guardian white')
* New wings (leather_green, leather_white, leather_blue)
* Adjusted tail (dragon, mouse)
* Typo fix on average balls
* Adjusted dragon tail to have more draconic design

names.gd
* Added names for koboldmale, koboldfemale, lizardfolkmale, lizardfolkfemale, and reptiliansurnames.
* Added Whims names for all the others.

races.gd
* Dragonkin
    * Added new wings.
    * Added scale color.
* New race, Gnoll
    * Stat array is 5, 4, 2, 5.
    * Racial ability is to hunt when finished with jobs that are outside the mansion.
* New race, Ogre
    * Stat array is 7, 2, 1, 7
    * Racial ability is increased bonus health, bonus health per point of endurance.
* New race, Giant
    * Stat array is 9, 2, 3, 5
    * Racial ability is that they consume 3x as much food.
* New race, Lizardfolk
    * Stat array is 5, 4, 3, 5
    * Racial ability is naturally armored scales.
* New race, Kobold
    * Stat array is 2, 6, 3, 5
    * Racial ability are they are more sensitive than other races when having sex.
* New race, Avali
    * Stat array is 1, 8, 8, 3
    * Racial ability is that they are more effective nurses.
* New race, Beastkin Mouse
   * Stat array is 2, 6, 5, 2
   * Racial ability is to pick locks on chests without lockpicks, and are more effective at picking with lockpicks.
* New race, Beastkin Squirrel
    * Stat array is 3, 5, 3, 5
    * Racial ability is that they carry 15 more units when in the party.
* New race, Beastkin Otter
    * Stat array is 4, 4, 2, 6
    * Racial ability is that they make friends faster.
* New race, Beastkin Bird
    * Stat array is 2, 5, 3, 6
    * Racial ability is that they gain more speed when flying compared to other races.

combat.gd
* Added logic to check for warrior, dancer, and mage specs for respective gains.
* Added logic to check for scaling weapon attributes and damage (no weapons just uses strength as scale).

combatdata.gd
* Added lizardfolkpatrol in the marsh. They are fully capturable. (They might be a bit strong, might wanna tune what level they are when you capture them)
* Added marsh equipment.
* Added new weapons to various group equips.

effect.gd
* Added a warrior effect similar to bodyguard. (Damage and speed)

exploration.gd
* Added mousekin to frostford guards.
* Added new weapons to chest loot.
* Added logic to allow mousekin to lockpick chests even without lockpicks. It is a new button. Check it out.

explorationregions.gd
* Added lizardfolkpatrols to marsh.

joblist.gd
* Added new bodyshapes to be able to perform jobs.

mainmenu.gd
* Added a lotta dictionaries
* Made it so ear selection is no longer inherently disabled, 
* Logic added to process scale and feather colors.
* Added to the _ready function to check for applyCapitulizeTweaks.

newsexsystem.gd
* Added new categories to support new sex actions.
* Kobold logic stored here to make them more sensitive.
* Logic added for triple penetration. (Needs more testing)

person.gd
* Added var scalecolor
* Added var feathercolor
* Essences
    * Gnolls drop tainted.
    * Kobolds drop magic.
    * Lizardfolk drop nature.
* Added to var genealogy the new races.

slave_tab.gd
* Added logic to assign base stats before modification to slaves in mansion in the top bar.

statstab.gd
* Added logic to check for specialization based abilities.

traits.gd
* Changed devoted and slutty to instead adjust their max values rather than adding to their base. (Gameplay wise this creates a situation where a user would want to max the mental out first to gain maximum value, was dumb.)
* Added max values to each of the stat adjusting traits, in base game without use_capitulize_tweaks they are effectively untouched.

backwardscompatibility.gd
* Added some statements to hopefully enable new persons from old saves.

### CODE REVIEW

File: scripts/newsexsystem.gd [hunk 1]
```diff
@@ -369,8 +373,23 @@ class member:
 					if scene.scene.takerpart == '':
 						penistext += " {^semen:seed:cum} {^pours onto:shoots onto:falls to} the {^ground:floor} as [he1] ejaculate[s/1]."
 					elif ['anus','vagina','mouth'].has(scene.scene.takerpart):
-						if scene.scene.get('takerpart2') && scene.scene.givers.size() == 2 && scene.scene.givers[1] == self:
+						if (scene.scene.get('takerpart2') && scene.scene.givers[1] == self) || (scene.scene.get('takerpart3') && scene.scene.givers[2] == self): # /Capitulize
```
Comment (BurnerXD) [inline scripts/newsexsystem.gd]: Revert the logic on this back

File: scripts/sexdescriptions.gd [hunk 2]
```diff
@@ -0,0 +1,111 @@
+var C_FULL_BODY_SCALES = 'fullscales'
```
Comment (ankmairdor) [inline scripts/sexdescriptions.gd]: This mod already patches this file. Any changes should be performed there.

Comment (BurnerXD) [reply scripts/sexdescriptions.gd]: Fixed

File: scripts/newsexsystem.gd (hunk 1 above)
Comment (BurnerXD) [reply scripts/newsexsystem.gd]: Fixed

File: globals.gd [hunk 3]
```diff
@@ -1501,6 +1536,18 @@ func fertilize_egg(mother, father_id, father_unique):
 		elif father_unique == 'raccoon':
 			father.genealogy.raccoon = 100
 			father.race = 'Beastkin Tanuki'
+		elif father_unique == 'mouse':
+			father.genealogy.mouse = 100
+			father.race = 'Beastkin Mouse'
+		elif father_unique == 'squirrel':
+			father.genealogy.squirrel = 100
+			father.race = 'Beastkin Squirrel'
+		elif father_unique == 'otterr':
```
Comment (BurnerXD) [inline globals.gd]: god dammit

File: scripts/characters/names.gd [hunk 4]
```diff
@@ -10,7 +10,42 @@ var names = {
...
...oa","Spipine","Spiqys","Spiyore","Taci","Tagyra","Talys","Tamiphi","Tamyra","Taqaryn","Taqys","Tassa","Tawyse","Taza","Tazyre","Tifadysa","Tifakini","Tifala","Tifamila","Tifawyse","Tifaxi","Tridira","Trifi","Trigani","Triqaryn","Trissa","Trisys","Triyaris","Ufefyx","Ufegani","Ufenoa","Ufephina","Ufepine","Ufeqys","Ufetina","Ufexi","Urici","Urihani","Urili","Urilys","Urina","Uriqys","Uriroe","Urisany","Urissa","Urisys","Uriyore","Venbys","Venceli","Venci","Vengani","Venli","Venpine","Vobys","Vofyx","Vohana","Vokasys","Vola","Voli","Vona","Vophina","Vorhana","Vossa","Votina","Vowyn","Welbi","Welci","Welgani","Welhana","Welkasys","Wellin","Welmyn","Welna","Welpine","Welqys","Weltra","Wrola","Wromyn","Wroniana","Wropine","Wrosys","Wrowyn","Wroza","Xabi","Xabys","Xaceli","Xakasys","Xaphina","Xaroe","Xayaris","Xyrobi","Xyrogyra","Xyroli","Xyrolin","Xyroniana","Xyroqaryn","Xyrorhana","Xyroroe","Xyrosys","Xyrotina","Xyrowyn","Xyrozyre","Ylonove","Ylophina","Yloqys","Ylotina","Ylozyre","Yobys","Yolin","Yomiphi","Yorhana","Yoza","Zanifyx","Zanigani","Zanihana","Zanikini","Zanimyra","Zanisany","Zaniyaris","Zindysa","Zinhani","Zinla","Zinmila","Zinniana","Zinqaryn","Zintina","Zinwyn","Zinxi"],
```
Comment (BurnerXD) [inline scripts/characters/names.gd:19]: Credit to Whims on these in the discord, I'll make sure he gets that credit in the changelog.

File: globals.gd (hunk 3 above)
Comment (BurnerXD) [reply globals.gd]: Fixed

File: scripts/combatdata.gd [hunk 5]
```diff
@@ -102,6 +102,11 @@ var enemygrouppools = {
 		description = "You spot a local [color=aqua]patrol of elven warriors[/color] scouting the surroundings. Surely, they won't be happy with trespassers as this is elven territory. ", #Removed Tribal to avoid confusion
 		descriptionambush = 'You are attacked by a [color=aqua]small group of elven warriors[/color]. ',
 	},
+	lizardfolkpatrol = {
+		units = [['lizardfolkwarrior',2,4]], awareness = 15, captured = null, special = null,
+		description = "You spot a local [color=aqua]patrol of lizardfolk warriors[/color] surverying the marsh. They are very territorial, your transgression won't be happy with appreciated. ",
```
Comment (BurnerXD) [inline scripts/combatdata.gd]: Typo here

Comment (BurnerXD) [reply scripts/combatdata.gd]: Fixed

File: scripts/exploration.gd [hunk 6]
```diff
@@ -244,13 +251,95 @@ var treasuremisc = [['magicessenceing',7],['taintedessenceing',7],['natureessenc
...
+		outside.playergrouppanel()
+		###---End Exploration---###
+		treasurechestoptions(text)
+	else:
+		showlootscreen(text)
 
+func chestmouselockpick(person):
+	var unlock = false
+	var text = ''
+	person.energy -= 10
```
Comment (BurnerXD) [inline scripts/exploration.gd:323]: Swap this to 5

Comment (BurnerXD) [reply scripts/exploration.gd:323]: nvm

**Comment by AricTriton** [comment]
Holding on Merging until v1.8 to confirm there's no backwards compatibility issues (since there wasn't certainty on that, don't want to blow up save games on a previous buggy release)

**Pull request merged**